src/ducks/apps/components/ApplicationPage/Header.jsx

```diff
@@ -41,11 +43,22 @@ export const Header = ({
     openApp(webviewIntent, app)
   }
 
-  const openConnector = () => {
+  const openConnector = async () => {
     if (isFlagshipApp()) {
       return webviewIntent.call('openApp', app.related, app)
     } else {
       const intents = new Intents({ client })
+      try {
+        // `intents.createService` throw an error if executed outside of an Intent iframe or if it is called without intentId.
+        const service = await intents.createService()
+
+        if (intentData.data.terminateIfInstalled) {
+          return service.terminate(app)
+        }
+      } catch (error) {
+        log('info', error, 'openConnector')
```

Comment:
are you sure about the use of cozy-logger? 

because someone recently added: `It should only be used for connectors and services. For webapps, prefer [cozy-minilog](https://github.com/cozy/cozy-libs/tree/master/packages/cozy-minilog) for the following reasons:` to cozy-logger's readme

Reply:
Tout à fait, je ne l'avais pas mis volontairement car avec cette approche, nous passions dans le catch à tous les coup lors du click sur le bouton "Ouvrir" dans l'app cozy-store (hors Intent).

J'ai revu l'approche, en conditionnant le try/catch (uniquement dans un Intent), de ce fait le catch doit être correctement remonté. J'ai donc ajouté cozy-minilog dans un dernier commit 👍